tests/jest/unit/core/experiments/assets/js/admin/behaviors/experiments-dependency.test.js

```diff
@@ -0,0 +1,112 @@
+import ExperimentsDependency from 'elementor/core/experiments/assets/js/admin/behaviors/experiments-dependency';
```

Comment:
Please test the case where the user sets the value to `default` and the default is `inactive`

Reply:
Good idea. Done.